x-pack/plugins/stack_connectors/server/connector_types/crowdstrike/crowdstrike.test.ts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CrowdstrikeConnector } from './crowdstrike';
+import { actionsConfigMock } from '@kbn/actions-plugin/server/actions_config.mock';
+import { loggingSystemMock } from '@kbn/core-logging-server-mocks';
+import { actionsMock } from '@kbn/actions-plugin/server/mocks';
+import { CROWDSTRIKE_CONNECTOR_ID } from '../../../public/common';
+
+const tokenPath = 'https://api.crowdstrike.com/oauth2/token';
+const hostPath = 'https://api.crowdstrike.com/devices/entities/devices/v2';
+const actionsPath = 'https://api.crowdstrike.com/devices/entities/devices-actions/v2';
+describe('CrowdstrikeConnector', () => {
+  const connector = new CrowdstrikeConnector({
+    configurationUtilities: actionsConfigMock.create(),
+    connector: { id: '1', type: CROWDSTRIKE_CONNECTOR_ID },
+    config: {},
+    secrets: { clientId: '123', clientSecret: 'secret' },
+    logger: loggingSystemMock.createLogger(),
+    services: actionsMock.createServices(),
+  });
+  let mockedRequest: jest.Mock;
+
+  beforeEach(() => {
+    // @ts-expect-error private static - but I still want to reset it
+    CrowdstrikeConnector.token = null;
```

Comment:
This is interesting (surprising too).  I guess the transpiler for TS to JS does not define the property as "private" class member, thus why you are able to do this

Reply:
🤷  :)